### DISCUSSION TITLE
Export of library to set of svg symbols

### DISCUSSION THREAD

**Discussion by bpfel** [jgraph/drawio#4136] [General]
* [x] I agree to follow the [Code of Conduct](https://github.com/jgraph/drawio/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x] I have searched the issue tracker for a feature request that matches the one I want to file, without success.

**Is your feature request related to a problem? Please describe.**
I have composed a custom library. The symbols in it are used in a diagram, to which I'd like to add a symbol legend. For that I to export each and every symbol one by one.

**Describe the solution you'd like**
I would like to be able to export all symbols in a library as single .svg files.

**Describe alternatives you've considered**
I have considered parsing the `.drawio` file containing the library and converting the symbols one by one. A possible approach is described [here](https://inventwithpython.com/blog/2022/08/25/how-to-run-pip-from-the-python-interactive-shell/). What's missing is a piece of code that actually does the conversion from `mxGraphModel` to `svg`.

I assume that functionality must exist. So if there is an accessible (python or similar) version of mxGraphModel2svg-conversion is available I would be glad for every hint!

**Comment by alderg** [comment]
Try the [desktop app](https://get.draw.io) CLI, eg. on macOS `/Applications/draw.io.app/Contents/MacOS/draw.io -x -f svg Untitled\ Diagram.drawio` (use `--help` for usage)
